README.md

```diff
@@ -29,6 +29,7 @@ The Neurobagel API is a REST API, developed in [Python](https://www.python.org/)
 - [Testing](#testing)
 - [License](#license)
 
+Please refer to our [**official documentation**](https://neurobagel.org/api/) for more information on how to use the API.
```

Comment:
🍒 : I think this line may look/work better if placed above the shortcut bullet points. Wdyt?